panther_driver/src/driver_node.py

```diff
@@ -170,7 +173,13 @@ def __init__(self, name: str) -> None:
         if self._publish_odom:
             self._odom_msg = Odometry()
             self._odom_msg.pose.covariance = [0.1 if (i % 7) == 0 else 0.0 for i in range(36)]
-            self._odom_msg.twist.covariance = [0.1 if (i % 7) == 0 else 0.0 for i in range(36)]
+
+            twist_cov = [0.0] * 36
+            cov_value = [self._v_x_var, self._v_y_var, 1e6, 1e6, 1e6, self._v_yaw_var]
+            for i in range (len(cov_value)):
+                twist_cov[7*i] = cov_value[i]
```

Comment:
Looking on pose covariance I decide to live it in that way. Now is more similar. It could be also like this:
```
cov_value = [self._v_x_var, self._v_y_var, 1e6, 1e6, 1e6, self._v_yaw_var]
twist_cov = [cov_value[i//7] if (i % 7) == 0 else 0.0 for i in range(36)]
```
bur this is not intuitive.

Reply:
I decide to assign to full 6x6 matrix